pkg/destroy/ovirt/destroyer.go

```diff
@@ -33,27 +33,31 @@ func (uninstaller *ClusterUninstaller) Run() error {
 	}
 	defer con.Close()
 
-	err = uninstaller.removeVms(con)
-	uninstaller.Logger.Errorf("Removing VMs - error: %s", err)
-	err = uninstaller.removeTag(con)
-	uninstaller.Logger.Errorf("Removing Tag - error: %s", err)
-	err = uninstaller.removeTemplate(con)
-	uninstaller.Logger.Errorf("Removing Template - error: %s", err)
+	if err := uninstaller.removeVMs(con); err != nil {
+		uninstaller.Logger.Errorf("Failed to remove VMs: %s", err)
```

Comment:
nit: error message strings should begin with lowercase letters as they are often appended

Reply:
I agree (I think), but this isn’t an error string. This is a log line, which I generally capitalize, with the error string  embedded (and that string should be lowercase).

Reply:
Correct. My mistake!